### ISSUE TITLE
GPS module neo-6 does not connect.

### ISSUE THREAD

**Issue by Apnet66** [ThisIsJustARandomGuy/dobson-star-tracker#7]
Everything works from the Arduino ide monitor.  no data from GPS module, connected to serial1.

**Comment by sebaw1** [comment]
I had the same problem. For a long time I didn't receive any coordinates. I had to put gps module near window(when indoor) or go outside.

**Comment by Apnet66** [comment]
does it work with Stellarium?  I don't have synchronization.

**Comment by sebaw1** [comment]
I haven't finished my project yet. I just connected GPS module to arduino nano and uploaded simple program to get coordinates using FuGPS library

**Comment by Apnet66** [comment]
Can you send me a sketch with this library? For Nano?

**Comment by sebaw1** [comment]
https://github.com/fu-hsi/FuGPS/blob/master/examples/FuGPS_Usage/FuGPS_Usage.ino

**Comment by Apnet66** [comment]
> https://github.com/fu-hsi/FuGPS/blob/master/examples/FuGPS_Usage/FuGPS_Usage.ino

Looks like a broken module, everything is "0"

**Comment by sebaw1** [comment]
No, I had the same problem. When i moved outside or just was closer to window, waited some time, even 20 minutes, and then module connected to GPS

**Comment by Apnet66** [comment]
> I haven't finished my project yet. I just connected GPS module to arduino nano and uploaded simple program to get coordinates using FuGPS library

Defeated GPS with an external antenna. Connected the Arduino Nano Serial 2 display (16,17 on Mega)
 In the photo in Thingeverse, you can see that the gearbox is mounted on a vertical stepper. they are 5.18 to 1. But the fight against Stellarium was not successful......
